tests/testAutopas/AutoPasTest.cpp

```diff
@@ -0,0 +1,16 @@
+/**
+ * @file AutoPasTest.cpp
+ * @author seckler
+ * @date 29.05.18
+ */
+
+#include "AutoPasTest.h"
+#include "AutoPas.h"
+
+TEST_F(AutoPasTest, getRegionParticleIterator) {
+  auto iter = autoPas.getRegionIterator({0., 0., 0.}, {4., 4., 4.});
+
+  for (; iter.isValid(); ++iter) {
+    iter->setR(iter->getR());
+  }
```

Comment:
Isn't there an assert missing here? So far this test seems to just test for crashes or am I missing s.th.? :D

Reply:
yes and it makes certain, that the method is actually called, so that potential template errors are found.
I have managed to compile the code, while this block was not there and adding that test let the compilation fail...

Reply:
Then at least document this here. The way it is now it looks a bit strange and superfluous.

Reply:
Then at least document this here. The way it is now it looks strange and superfluous.

Reply:
done